Tasks/PublishSymbolsV2/PublishSymbols.ts

```diff
@@ -0,0 +1,113 @@
+import * as fs from "fs";
+import * as uuidV4 from 'uuid/v4';
+import * as telemetry from "utility-common/telemetry";
+import * as clientToolUtils from "clienttool-common/ClientToolUtilities";
+import * as clientToolRunner from "clienttool-common/ClientToolRunner";
+import * as tl from "azure-pipelines-task-lib";
+import { IExecSyncResult, IExecOptions } from "azure-pipelines-task-lib/toolrunner";
+
+const symbolRequestAlreadyExistsError = 17;
+
+export async function run(clientToolFilePath: string): Promise<void> {
```

Comment:
Do we know if we also need to include the "Index sources" part when running with the new symbol.exe?

Reply:
From Carl - with the “precheck” it would download the net core version of symbol in the place of the framework version and then the powershell script would just use that netcore version without having to muck with the script at all

Reply:
I don't see the indexing code here - the netcore version will only publish symbol AFAIK.

Reply:
Tasks/PublishSymbolsV2/PublishSymbols.ps1 is the one that does the indexing of source code.